department_app/views/routes.py

```diff
@@ -1,5 +1,258 @@
 """Routes web-app"""
-from flask import Blueprint
+from sqlalchemy.exc import IntegrityError
+from flask_restful import abort
+from flask import Blueprint, render_template, request, redirect, url_for
 
+from department_app.service import CRUDDepartment, CRUDAddress, CRUDSkill, CRUDEmployee, CRUDLocation, CRUDPermission
+from department_app.views.forms.forms import DepartmentForm, SkillForm, AddressForm, LocationForm, \
+    PermissionForm, EmployeeForm
+from department_app.models import DepartmentModel, PermissionModel, AddressModel, LocationModel,\
+    SkillModel, EmployeeModel
 
 frontend = Blueprint('frontend', __name__)
+"""
+ADD TEMPLATES
+"""
+
+
+@frontend.route('/add/address', methods=['POST', 'GET'])
```

Comment:
I'd use class based views instead of simple functions. It'd be nice to describe each view in separate file